Projects/Server/Maps/MapSelection.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Server.Maps;
+
+public class MapSelection
+{
+    private MapSelectionFlags _selectedMaps;
+    private static readonly Array _mapSelectionValues = Enum.GetValues(typeof(MapSelectionFlags));
```

Comment:
Let's avoid `Array` because it is a legacy type. Instead use `MapSelectionFlags[]` so it forces the compiler to use `Enum.GetValues<T>()` :)

Reply:
ooo alright! :)

Reply:
ahh, Enum.GetValues(typeof(MapSelectionFlags)) returns an Array, but Enum.GetValues<MapSelectionFlags>() returns MapSelectionFlags, cool :+1: